lib/resources/oidc.js

```diff
@@ -13,7 +13,14 @@
 // OpenID Connect auth handling using Authorization Code Flow with PKCE.
 // TODO document _why_ auth-code-flow, and not e.g. implicit flow?
 
-const { generators } = require('openid-client');
+const { // eslint-disable-line object-curly-newline
+  authorizationCodeGrant,
+  buildAuthorizationUrl,
+  calculatePKCECodeChallenge,
+  fetchUserInfo,
+  randomPKCECodeVerifier,
+  randomState,
+} = require('openid-client'); // eslint-disable-line object-curly-newline
```

Comment:
this is a nit / preference: we can `const client = require('openid-client');` then the rest of the code in this file will be easier to read and understand, specially knowing which functions are coming from the external library.